config.xml

```diff
@@ -2,7 +2,7 @@
 <module>
 	<name>cronjobs</name>
 	<displayName><![CDATA[Cron tasks manager]]></displayName>
-	<version><![CDATA[1.3.2]]></version>
+	<version><![CDATA[1.4.0]]></version>
```

Comment:
It's best to leave the version bump to the moment we merge on master

Reply:
Fixed in https://github.com/PrestaShop/cronjobs/pull/43/commits/17c3dfe202487323a51ff0c177f505358c0334e4